core-java-modules/core-java-io-2/src/main/java/com/baeldung/listfiles/ListFiles.java

```diff
@@ -27,31 +26,31 @@ public Set<String> listFilesUsingJavaIO(String dir) {
     public Set<String> listFilesUsingFilesList(String dir) throws IOException {
         try (Stream<Path> stream = Files.list(Paths.get(dir))) {
             return stream
-                .filter(file -> !Files.isDirectory(file))
-                .map(Path::getFileName)
-                .map(Path::toString)
-                .collect(Collectors.toSet());
+                    .filter(file -> !Files.isDirectory(file))
+                    .map(Path::getFileName)
+                    .map(Path::toString)
+                    .collect(Collectors.toSet());
```

Comment:
This formatting change looks like it needs reverting

Reply:
I double-checked the formatting. The indentation is correct and follows the IntelliJ default formatting for functional code.